identity-service/src/notifications/index.js

```diff
@@ -187,15 +190,21 @@ class NotificationProcessor {
           owner: owners.tracks[x.trackId]
         }
       })
-      await indexNotifications(notifications, tx, audiusLibs)
+
+      await processNotifications(notifications, tx)
```

Comment:
let's add a small comment explaining the separation of concerns between these two functions as well as the ones below